fds/services/dvc_service.py

```diff
@@ -18,8 +20,15 @@ class DvcChoices(Enum):
     ADD_TO_DVC = "Add to DVC"
     ADD_TO_GIT = "Add to Git"
     IGNORE = "Ignore"
+    SKIP = "Skip"
     STEP_INTO = "Step Into"
 
+@dataclass
+class AddToDvc:
+    file_to_add: Optional[str]
+    file_to_ignore: Optional[str]
+    file_to_skip: Optional[str]
+
```

Comment:
Seems pretty confusing that you have another dataclass which is very similar to `DvcAdd`, I had to double check that this isn't a copy-paste mistake.
Not critical, but it might just be simpler and less confusing to keep using tuple return values - after all, this is just local private function calls, so having an official type as a declared interface is less important

Reply:
Yeah but the tuple was very hard to use, and i was skeptical that i would miss something with the tuple like basically interchange the places which can cause us pain

Reply:
You can also think of this like an internal dataclass, so its not exposed outside